tables_en/2-02.csv

```diff
@@ -1,22 +1,20 @@
 notation,path,name,description
-ABOS,\WIGOS\GOS\GOS Other elements\ABOS,ABOS,Aircraft-Based Observing Systems
 ACTRIS,\WIGOS\GAW\GAW Other elements\ACTRIS,ACTRIS,"Aerosol,Clouds and Trace Gases Research Infrastructure, www.actris.eu"
 ADD,\WIGOS\GOS\GOS Other elements\ABOS\ADD,ADD,Aircraft Derived Data (Mode-S/ADS-B)
-ADNet,\WIGOS\GAW\GAW Contributing networks\GALION\ADNet,ADNet,AD-Net Asian Dust and Lidar Observation Network
+ADNet,\WIGOS\GAW\GAW Contributing networks\ADNet,ADNet,AD-Net Asian Dust and Lidar Observation Network
 ADSC,\WIGOS\GOS\GOS Other elements\ABOS\ADS-C,ADS-C,Automatic Data Surveillance (ICAO). These are transmitted at selected points of the flightpath.
-AEROCAN,\WIGOS\GAW\GAW Other elements\AOD\AEROCAN,AEROCAN,"AEROCAN, federated to AERONET"
-AERONET,\WIGOS\GAW\GAW Other elements\AOD\AERONET,AERONET,AErosol RObotic NETwork
+AEROCAN,\WIGOS\GAW\GAW Other elements\AEROCAN,AEROCAN,"AEROCAN, federated to AERONET"

```

Comment:
@amilan17  Not sure if double-quotes should be removed or inserted everywhere else.

Reply:
The double quotes are in this line to ensure the integrity of the comma-delimited CSV. They are required when there is a comma within the text of a cell in the table. It is not necessary when there are no commas.

Reply:
@joergklausen - please resolve this conversation if you are satisfied with my response.